onnxruntime/contrib_ops/rocm/bert/skip_layer_norm.cc

```diff
@@ -106,7 +109,8 @@ Status SkipLayerNorm<T>::ComputeInternal(OpKernelContext* ctx) const {
       (bias != nullptr) ? reinterpret_cast<const HipT*>(bias->Data<T>()) : nullptr,
       epsilon_,
       hidden_size,
-      static_cast<int>(element_count));
+      static_cast<int>(element_count),
```

Comment:
This `static_cast` already indicate the typing for the downstream fucntion signature is not proper. Maybe switch to `size_t` alltogether.